traitsui/qt4/range_editor.py

```diff
@@ -693,6 +693,12 @@ class RangeTextEditor(TextEditor):
     #  Trait definitions:
     # -------------------------------------------------------------------------
 
+    #: Low value for the slider range
+    low = Any()
+
+    #: High value for the slider range
+    high = Any()
```

Comment:
I'm not sure why these are any traits. Is that because users can set `low` to a value or another trait?

Reply:
TBH II'm not sure either... I was simply following the `SimpleSliderEditor`.  The factory class has them as `Property`, whose setter makes use of this method:
https://github.com/enthought/traitsui/blob/7e1d9ed6e91e2fe4cc356f4966653a2e889909a4/traitsui/editors/range_editor.py#L169-L176